doc/book/usage.md

```diff
@@ -87,6 +87,13 @@ $request = Zend\Diactoros\ServerRequestFactory::fromGlobals(
 );
 ```
 
+When no cookie array is supplied, `fromGlobals` will first try to parse the supplied `cookie` header
+before falling back to the `$_COOKIE` superglobal. This is done because PHP has some legacy handling
+for request parameters which were then registered as global variables. Due to this, cookies with a period
+in the name were renamed with underlines. By getting the cookies directly from the cookie header, you have
+access to the original cookies in the way you set them in your application and they are send by the user
+agent.
```

Comment:
Question: what happens if `$_COOKIE` is passed? `fromGlobals()` allows passing an array of cookies, and some of our documentation actually demonstrates passing in the superglobals explicitly. Should we discourage that behavior when it comes to cookies?

If so, we should likely add a note to that effect, and change some of the examples.